Jetsnack/app/src/main/java/com/example/jetsnack/ui/components/QuantitySelector.kt

```diff
@@ -47,17 +45,15 @@ fun QuantitySelector(
     ConstraintLayout(modifier = modifier) {
         val (qty, minus, quantity, plus) = createRefs()
         createHorizontalChain(qty, minus, quantity, plus, chainStyle = ChainStyle.Packed)
-        ProvideEmphasis(emphasis = AmbientEmphasisLevels.current.medium) {
```

Comment:
You removed the medium emphasis? Intentional?

Reply:
Sharp! That's a copy-paste mistake in a previous commit. I'll fix it!

Reply:
![](https://media1.giphy.com/media/3ohzdQ3TjcSdJqQa52/giphy.gif)

Reply:
A-ha! This is where it went wrong. The auto-replace replaced it with "high" emphasis, and I missed it. Later on I removed all the redundant "high" providers.

<img width="700" alt="Screen Shot 2020-11-11 at 10 45 00" src="https://user-images.githubusercontent.com/6952116/98796036-38bbbd00-240b-11eb-9634-238d41b6cf18.png">

It's fixed now!

Reply:
Ah yeah I noticed the same. I'll raise a bug because the `ReplaceWith` could definitely be improved.

Reply:
https://issuetracker.google.com/172989644